### ISSUE TITLE
TC E-vote Comment 2: indicate other spatio-temporal data as well as environmental by renaming

### ISSUE THREAD

**Issue by chris-little** [opengeospatial/ogcapi-environmental-data-retrieval#206]
`Consider renaming the standard, as the term 'environmental'  does not seem to cover the scope. Any spatio-temporal data could be exposed using an EDR API. As I see it, it provides a way to find and  retrieve spatio-temporal data using different subsetting techniques, making it convenient for users to select the subset of the data they need.`

**Comment by chris-little** [comment]
The [EDR API SWG meeting 43, on 18 Feb 2021](https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/wiki/2021-02-18) discussed renaming of the standard and agreed that renaming was a decision for the OGC Planning Committee, as it would involve not just the SWG members in significant work, but also OGC Staff, in revising numerousnnames, links and also marketing material.

**Comment by chris-little** [comment]
The OGC Architecture Board also discussed the issue briefly, and there was sympathy for not renaming, and some strong support for the existing EDR name. @chris-little and @ogcscotts would prepared briefing material for the Planning Committee in March 2021.

**Comment by dr-shorthair** [comment]
What was the case for not-renaming? This is not yet an adopted standard, so now is the time to make sure the name reflects the scope. Can you summarize please.

**Comment by dr-shorthair** [comment]
(Stu Weibel of Dublin Core fame once noted that it was best to 'eat crow early, while it is young and tender')

**Comment by chris-little** [comment]
cCmments from @tomkralidis on previous issue #199 
These (renames) would probably entail changing all of the links in OGC registeries and the document.
Rename suggestions:
STDR   - Spatio-Temporal Data Retrieval;  (this might cause some confusion with STAC proper)
STSDR - Spatio-Temporal Simple Data Retrieval;
STDE   - SpatioTemporal Data Extraction.

**Comment by chris-little** [comment]
Issues #206, #207, #215, #216, and #221 arising form the TC E-vote Comments are basically the same.

**Comment by chris-little** [comment]
Addressed by PR #239

**Comment by chris-little** [comment]
#221 has a useful summary:
`The argumentation brought for maintaining the term “Environmental” led to reflection on common usages of the term, think we’re rotating around the following meanings of the term “Environmental”:

"One’s surroundings, implicit in the Geospatial nature of OGC.
The physical, chemical, and biotic factors one associates with environmental protection and sciences (e.g. Environment Agencies (EPA, EEA))."

The first one is mostly redundant with the geospatial domain, the second is what can be described with the O&M standard. To our worries on the “Environmental” label -if EDR goes out as "Environmental Data Retrieval", this will most likely seriously set back environmental data provision due to the following:

Most environmental data requires provision of measurement metadata for correct interpretation, e.g. air quality, water quality, soil data, biodiversity. This is the reason the O&M standard was initially developed. To our experience, the two domains that can do without are meteorology and marine, which has mostly to do with organizational background.
If OGC propagates an "Environmental Data Retrieval" API, many administrators in environmental agencies`

**Comment by dr-shorthair** [comment]
The previous comment is truncated. The [last sentence continues](https://github.com/opengeospatial/ogcapi-environmental-data-retrieval/issues/221#issue-815615760)
> many administrators in environmental agencies will insist on using it, while not understanding its lack of suitability. To our view, this would be a sad legacy for a well meant standard.

I agree with the concern and strongly recommend that the standard be renamed so as not to create the wrong impression in environment agencies.

**Comment by EmDevys** [comment]
> Comments from @tomkralidis on previous issue #199 and others.
> These (renames) would probably entail changing all of the links in OGC registeries and the document.
> Rename suggestions:
> STDR - Spatio-Temporal Data Retrieval; (this might cause some confusion with STAC proper)
> STSDR - Spatio-Temporal cl;
> STDE - Spatio-Temporal Data Extraction.
> STRR - Spatio-Temporal Resource Retrieval
> STRSR - Spatio-Temporal Resource Simple Retrieval
> STFR - Spatio-Temporal Feature Retrieval
> 
> SDR - Spatiotemporal Data Retrieval
> MDR - Multidimensional Data Retrieval

As this candidate standard is not yet approved (though the ballot got quorum) and there are key issues with the perverse use of Environmental data (which it is not in the common acceptance of this term), every effort should be done by the authors, with the help of the SWG and the OGC to clarify the naming of this (simple) API for retrieval of features at a Position, Radius, Area, Cube, Trajectory, Corridor, Items, Locations, Instances (sampling) (many of them apparently being coverage features, but the draft specification also provides an example with Observation).
As @dr-shorthair reminds it, better to 'eat crow early, while it is young and tender'.
The draft document under the github has evolved positively by getting rid (or nearly) of the term / concept of "environmental data resource", which was strongly discussed and would have involved some associated data model, which is not the intent of the authors.
Let's reflect this in the naming of this API specification, which is quite flexible on the data it is supposed to address. Updating the document and supposedly OGC registries (which should be populated on basis of adopted standards and not draft specifications under ballot is some kind of logistics which I am aware being some kind of burden (having done such kind of works), but this is nothing compared to the trouble that a bad naming for this new generation API may create in the future, to users, communities, and OGC.
For the names, probably the authors and the SWG should probably vote. Spatio-Temporal Data or Feature or Resource are acceptable to me (but not Environmental once again); Retrieval or extraction as well. As this is supposed to be a simple and flexible API (compared to the other more complex OGC API), it seems a good idea to add this in the naming. But not "Simple Data Retrieval" because the data is  not that simple ...

**Comment by chris-little** [comment]
Please see the second comment on this issue.

The owners of the several implementations have now been prepared to change the name (probably to 'spatiotemporal` rather than 'environmental') depending on the Planning Committee's decision in two weeks time.

We have noted that OGC and ISO do not have a definition of spatiotemporal data (resource), only a "spatiotemporal object", which we have modified appropriately.

**Comment by EmDevys** [comment]
An additional comment on this ambiguous use of "Environmental data" term: 
"environmental data" is a term with an ISO definition as follows:
integrated set of data elements, each describing some aspect of an environment and the elements or events expected there
ISO/IEC 18023-1:2006 Information technology — SEDRIS — Part 1: Functional specification, 3.1.7 
This definition is closer to the general usage of environmental data usage, though SEDRIS is mostly aimed at simulation.

**Comment by EmDevys** [comment]
And in order to ensure this comment is appropriately solved, if the proposal to adjust the name / title of this API, whose intent does not seem to define (or redefine) "Environmental data" - hopefully not as far as I could understand from the EDR API SWG meeting on 26th March, it seems that the current golden draft available on 26 MArch still contains several occurences of the "Environmental data" term, for terms such as "Environmental data collection(s)" and "Environmental Data API Paths" (Table 2). In case the change of naming / title is confirmed, the editors should take care of changing these terms appropriately.

**Comment by chris-little** [comment]
@EmDevys Agreed.

**Comment by chris-little** [comment]
@EmDevys The OGC Planning Committee voted unanimously to not change the name. 

All occurrences of "environmental data resources" and similar phrases have been changed to "spatio-temporal" resources and similar, with the addition of an ISO based definition of spatio-temporal data. No definition of 

As the standard is now being frozen, we propose to close this issue as resolved

**Comment by EmDevys** [comment]
I have to agree with the decision of the OGC PC, and I hope the explanation of the title and the limitation of this ambiguous semantics will be clear enough and sufficient for avoiding any further trouble or mess.
@chris-little By the way, there is an unfinished sentence "No definition of" in the previous comment

**Comment by chris-little** [comment]
@EmDevys I've finished my thought above ;-)
